DOCSUP-1380 Document the max_server_memory_usage_to_ram_ratio setting (#142)
BayoNet/ClickHouse: docs/en/operations/server-configuration-parameters/settings.md

**Additional Info**

The default `max_server_memory_usage` value is calculated as `memory_amount * max_server_memory_usage_to_ram_ratio`.

**See also**

-   [max\_memory\_usage](../../operations/settings/query-complexity.md#settings_max_memory_usage)
-   [max_server_memory_usage_to_ram_ratio](#max_server_memory_usage_to_ram_ratio)

## max_server_memory_usage_to_ram_ratio {#max_server_memory_usage_to_ram_ratio}

> Defines the fraction of total physical RAM amount, available to the Clickhouse server. If the server tries to utilize more, the memory will be cut down to the appropriate amount. 

Reviewer: ```suggestion
Defines the fraction of total physical RAM amount, available to the Clickhouse server. If the server tries to utilize more, the memory is cut down to the appropriate amount. 
```

Possible values:

-   Positive double.

> -   0 — The Clickhouse server can use all available physical RAM amount.

Reviewer: ```suggestion
-   0 — The Clickhouse server can use all available RAM.
```
